### DISCUSSION TITLE
The fscrawler container is closing

### DISCUSSION THREAD

**Discussion by lefor** [dadoonet/fscrawler#1627] [Q&A] [answered]
The problem is that when running Docker on the latest version 2.10, I encounter these errors. 
![photo_2023-03-23_00-05-15](https://user-images.githubusercontent.com/33153596/227177598-ec738813-b6a7-4a38-9684-ad1399e4536b.jpg)
And the fscrawler container is closing.
![photo_2023-03-23_00-07-06](https://user-images.githubusercontent.com/33153596/227177664-878120ef-8dfe-46c6-9b51-4407f9ee328d.jpg)

**Comment by dadoonet** [comment, marked as answer]
Could you share exactly what command you are running, the configuration files and on which OS?

**Reply by lefor** [reply]
Windows 11 x64 and Windows 10 x64 
Docker Desktop v14.17.1
open Windows PowerShell
git clone https://github.com/dadoonet/fscrawler.git
cd C:\Users\USERNAME\Desktop\fscrawler\contrib\docker-compose-example-elasticsearch
docker compose up

**Reply by lefor** [reply]
I solved this issue by renaming the folder! The folder name had a space in it and that's why it was causing an error. After changing the folder name, everything was okay.

**Reply by dadoonet** [reply]
Amazing! I think this should may be documented or fixed.
